### PR TITLE
Added export functionality for Keras

### PR DESCRIPTION
**Pull request by utsavgarg**
* Added functions to calculate output shapes for Keras export
* Added Data, Conv, Pool, Activation, Dense, Flatten, Dropout layers
* Tested export of VGG-16

### CODE REVIEW

File: ide/utils/shapes.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        elif(net[layerId]['info']['type'] in ['InnerProduct', 'Recurrent', 'RNN', 'LSTM', 'Embed']):
+            net[layerId]['shape']['output'] = output(net[layerId])
+
+        elif(net[layerId]['info']['type'] == 'Flatten'):
+            net[layerId]['shape']['output'] = flatten(net[layerId])
+
+        elif(net[layerId]['info']['type'] == 'Reshape'):
+            net[layerId]['shape']['output'] = reshape(net[layerId])
+
+        elif(net[layerId]['info']['type'] in ['SPP', 'Crop']):
```
Comment (virajprabhu) [inline ide/utils/shapes.py]: We should throw a "This layer is not supported in Keras" error on the front end for this.

File: keras_app/views/layers_import.py [hunk 2]
```diff
@@ -12,9 +12,9 @@ def Input(layer):
 # ********** Vision Layers **********
 def Convolution(layer):
     params = {}
-    params['kernel_w'], params['kernel_h'] = layer.kernel_size
-    params['stride_w'], params['stride_h'] = layer.strides
-    params['pad_w'], params['pad_h'] = get_padding(params['kernel_w'], params['kernel_h'],
+    params['kernel_h'], params['kernel_w'] = layer.kernel_size
```
Comment (virajprabhu) [inline keras_app/views/layers_import.py]: Was this a mistake in the earlier PR?

**Comment by utsavgarg** [comment]
The `layers.py` was already renamed to `layers_import.py` in this PR. I have tested VGG-16, Resnet-50 and Inception-v3.

File: keras_app/views/layers_import.py (hunk 2 above)
Comment (utsavgarg) [inline keras_app/views/layers_import.py]: yes

File: ide/utils/shapes.py (hunk 1 above)
Comment (utsavgarg) [inline ide/utils/shapes.py]: Okay will add exception handling.

File: ide/static/bundle/bundle.js [hunk 3]
```diff
@@ -27760,7 +27760,7 @@
 	          delete netData[layerId].state;
 	        });
```
Comment (utsavgarg) [inline ide/static/bundle/bundle.js]: Not sure what you meant here

Comment (virajprabhu) [inline ide/static/bundle/bundle.js]: I meant we should add bundle.js to gitignore.

File: ide/utils/shapes.py (hunk 1 above)
Comment (virajprabhu) [inline ide/utils/shapes.py]: Sounds good.

**Comment by virajprabhu** [comment]
Great, please add everything that you've tested to the examples folder. Would be good to make subfolders for each framework.

**Comment by virajprabhu** [comment]
LGTM

**Pull request merged**